src/scripts/_types.ts

```diff
@@ -9,3 +9,31 @@ export const gameLevels: GameLevels = {
 	medium: 12,
 	hard: 18,
 };
+
+export interface ScoreBoardResults {
+	attemptCounter: number;
+	maxAttempts: number;
+	playerScore: number;
+	computerScore: number;
+}
+
+export const scoreBoardResults: ScoreBoardResults = {
+	attemptCounter: 0,
+	maxAttempts: 2,
+	playerScore: 0,
+	computerScore: 0,
+};
+
+export interface EventElementsValues {
+	clickedSrcValue: string | null;
+	srcToFindValue: string | null | undefined;
```

Comment:
undefined to po prostu wartość srcToFindValue z pytajnikiem na końcu (jesli moze byc taki przypadek, ze obiekt przyjdzie bez tej wartosci)